### PR TITLE
NO-ISSUE: increase memory requests and limits, and parametrize them

### PR DESCRIPTION
**Pull request by rccrdpccl**
Memory consumption is currently reaching somewhere shy of 1.6Gi in production (stablizes at this value).
Parametrizing this values would allow us to change it directly from CD when consumption changes, without the need of merging PR within the codebase

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @osherdp 
/cc @oourfali 

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by oourfali** [comment]
I guess we need to check with APP-SRE whether the cluster we're running on can indeed supply the memory and cpu request (well, CPU doesn't change here, just parameterized, but the memory is).

**Comment by rccrdpccl** [comment]
> I guess we need to check with APP-SRE whether the cluster we're running on can indeed supply the memory and cpu request (well, CPU doesn't change here, just parameterized, but the memory is).

Sounds good, in fact I think we should lower requests for CPU (on average we use 100m) - I was planning to have separate PR in the future to change CPU limits and include requests change there.

File: openshift/template.yaml [hunk 1]
```diff
@@ -117,6 +117,18 @@ parameters:
 - name: ENABLE_ORG_TENANCY
   value: "false"
   required: false
+- name: MEMORY_LIMIT
+  value: "2.4Gi"
+  required: true
```
Comment (gamli75) [inline openshift/template.yaml:122]: please change it to false

File: openshift/template.yaml [hunk 2]
```diff
@@ -117,6 +117,18 @@ parameters:
...
+- name: CPU_LIMIT
+  value: "500m"
+  required: true
```
Comment (gamli75) [inline openshift/template.yaml:125]: please change it to false

File: openshift/template.yaml [hunk 3]
```diff
@@ -117,6 +117,18 @@ parameters:
...
+- name: MEMORY_REQUEST
+  value: "1.6Gi"
+  required: true
+- name: CPU_REQUEST
+  value: "300m"
+  required: true
```
Comment (gamli75) [inline openshift/template.yaml:131]: please change it to false

File: openshift/template.yaml [hunk 4]
```diff
@@ -117,6 +117,18 @@ parameters:
...
   required: false
+- name: MEMORY_LIMIT
+  value: "2.4Gi"
+  required: true
+- name: CPU_LIMIT
+  value: "500m"
+  required: true
+- name: MEMORY_REQUEST
+  value: "1.6Gi"
+  required: true
```
Comment (gamli75) [inline openshift/template.yaml:128]: please change it to false

Comment (rccrdpccl) [reply openshift/template.yaml]: oops, my bad

**Review by osherdp — APPROVED** [review]

**Pull request merged**